tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
 	"compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```

Comment:
Oh, I see. I thought I was commenting on this PR, but it looks like the git history is messed up and merged some of your work into this 👍